test/stablecoin_converter.js

```diff
@@ -433,12 +432,13 @@ contract("StablecoinConverter", async (accounts) => {
 
       await waitForNSeconds(BATCH_TIME)
 
-      await stablecoinConverter.submitSolution(batchIndex + 1, owner, orderId, volume, prices, tokenIdsForPrice, { from: solutionSubmitter })
+      const volume2 = [2000, feeSubtracted(2000) * 2]
+      await stablecoinConverter.submitSolution(batchIndex + 1, owner, orderId, volume2, prices, tokenIdsForPrice, { from: solutionSubmitter })
 
-      assert.equal((await stablecoinConverter.getBalance.call(user_1, feeToken.address)).toNumber(), basicTrade.deposits[0].amount - 2 * getSellVolume(volume[0], prices[0], prices[1]), "Sold tokens were not adjusted correctly")
-      assert.equal((await stablecoinConverter.getBalance.call(user_1, erc20_2.address)).toNumber(), 2 * volume[0], "Bought tokens were not adjusted correctly")
-      assert.equal((await stablecoinConverter.getBalance.call(user_2, erc20_2.address)).toNumber(), basicTrade.deposits[1].amount - 2 * getSellVolume(volume[1], prices[1], prices[0]), "Sold tokens were not adjusted correctly")
-      assert.equal((await stablecoinConverter.getBalance.call(user_2, feeToken.address)).toNumber(), 2 * volume[1], "Bought tokens were not adjusted correctly")
+      assert.equal((await stablecoinConverter.getBalance.call(user_1, feeToken.address)).toNumber(), basicTrade.deposits[0].amount - getSellVolume(volume[0], prices[0], prices[1]) - getSellVolume(volume2[0], prices[0], prices[1]), "Sold tokens were not adjusted correctly")
```

Comment:
getSellVolume(volume[0]+volume2[0], prices[0], prices[1]) would be a little bit more succinct compared to these two function calls

Reply:
Yes, I can do that, but this wouldn't hold if we had used different prices for the second solution submission.

Reply:
Also, I am not entirely sure this is true since `getSellVolume` uses integer division.  Perhaps in this example it would be true, but I think I can construct valid examples that would violate this claim without having different prices. 

```js

function getSellVolume(x, priceNumerator, priceDenominator) {
  return Math.floor(Math.floor(x * priceDenominator / (feeDenominator - 1)) * feeDenominator / priceNumerator)
}
```